### ISSUE TITLE
CMake Macros are no longer installed since #6369

### ISSUE THREAD

**Issue by ccvca** [open62541/open62541#6402]
The file `open62541Macros.cmake` is no longer installed and included when using the project via `find_package`

Was this this intended in #6369 ?

Before this pull request, this was possible, like shown in the examples and the docs.

```
find_package(open62541 REQUIRED)
# [...]
ua_generate_nodeset_and_datatypes(
    NAME "di"
# [...]
```
Those where the relevant changes:

https://github.com/open62541/open62541/pull/6369/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL1563-L1569 

https://github.com/open62541/open62541/pull/6369/files#diff-da96f45d730b3404f73b2a9f2834433a8f46c5d1d2f538d1a6a27a098e5167aeL11

## Checklist

Please provide the following information:

 - [x] open62541 Version (release number or git tag): 1.4
 - [ ] Other OPC UA SDKs used (client or server): 
 - [ ] Operating system:
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

**Comment by basyskom-jvoe** [comment]
The python scripts and nodeset files required by the macros are also no longer installed.

**Comment by keba-estr** [comment]
@ccvca , @basyskom-jvoe : Should be resolved with PR #6406, please check if this meets your requirements.

**Comment by jpfr** [comment]
Fixed on 1.4

**Comment by ccvca** [comment]
Fixed by #6404 and #6434

To use the macros, import them like so:
```
find_package(open62541 REQUIRED)
include("${open62541_DIR}/open62541Macros.cmake") # <-- This line is required for the 1.4 branch
```
